lib/tasks/configNginx.js

```diff
@@ -145,13 +146,26 @@ function setupSSL(done) {
    // "ssl.pathKey ssl.pathCert"
    // "ssl.none"
    var sslOptions = Options.ssl || {};
+
+   /// handle any arguments that are in the form of "--ssl.arg"
    for (var o in Options) {
       var parts = o.split(".");
       if (parts[0] == "ssl") {
          sslOptions[parts[1]] = Options[o];
       }
    }
-
+   /// handle any arguments that are in the form of "[nginx.]sslArg"
```

Comment:
So ... does this actually catch ```Options["nginx.sslArg"] = 'value'``` parameters?

It looks like it would catch ```Options["sslArg"] ``` values, but not the ones prefixed with "nginx.".

Reply:
i'll double check the code flow here.  a bit fuzzy but I think we're trying to account for two situations -- the "new style" command line options with dots, and then the older style camel case only options.  I'll take a second look at this.

Reply:
OK, yes, this does work.  The reason why is because the program can be used in two ways:

1. 
`appbuilder ssl --self
`
2. 
```
appbuilder installV1 \
   --nginx.sslType=self \
   --nginx.sslCountryName=US
```

The "--nginx." prefix was already stripped prior to this.  The comment is meant to inform that it deals with the situation when passing it in with prefix, thus the square brackets.